test/MarketMaker.js

```diff
@@ -20,28 +20,21 @@ contract('marketMaker', function(accounts) {
 
   var mm;
   var token;
+  var crowdsale;
   var eventsWatcher;
 
   it("Create 24 months MM", async function() {
-    token = await help.simulateCrowdsale(100000000000, [40,30,20,10,0], accounts);
-    await help.checkToken(token, accounts, 100, [40,30,20,10,0]);
-
-    mm = await LifMarketMaker.new(
-      token.address, web3.eth.blockNumber+10, 100, 24,
-      accounts[1], {from: accounts[0]});
-
-    await mm.fund({value: web3.toWei(8, 'ether'), from: accounts[0]});
+    mmInitialBalance = 20000000;
```

Comment:
use `const` or `let`, the same for similar variables

Reply:
:+1: